setup/src/App.svelte

```diff
@@ -261,6 +273,48 @@
           </div>
         {/if}
 
+
+        <h3>Choose Registration Level</h3>
+        <div class="px-3 py-3">
+          <p>
+            You can configure the github app to register the runners at the
+            repository or organization level. <br>
+            The main difference are the
+            permissions assigned to the github app.<br>
+          </p>
+          <ul>
+            <li>
+              Repository level: the github app will
+              have full <b>administrative</b> access to the selected
+              repositories.
+            </li>
+            <li>
+              Organization level: the github app will have only the "self-hosted
+              github runner" permission to register runners in the organization. (Recommended for organizations)
+            </li>
+        </ul>
+          <div class="form-check">
+            <input
+              class="form-check-input"
+              type="radio"
+              bind:group={runnerLevel}
+              value="repo"
+              id="repo"
+            />
+            <label class="form-check-label" for="repo">Repository</label>
+          </div>
+          <div class="form-check">
+            <input
+              class="form-check-input"
+              type="radio"
+              bind:group={runnerLevel}
+              value="org"
+              id="org"
+            />
+            <label class="form-check-label" for="org">Organization</label>
```

Comment:
We may have to disable this option for user scoped apps.

Reply:
Good hint

Reply:
hmm thought about it - 
Technically it would be possible to have a user-scoped app that can be installed by multiple organizations. This would make only sense if it´s a public app.
Currently it`s restricted in the svelte app that only organization apps can be public.

From a usage standpoint. If you already create a app primarily for multiple organizations why you would use a user-scoped app for that. Makes no sense, because at some point you would want to test it in your organization at least.

So yes - I would propose to restrict it and exclude user scoped apps.
=> will implement this.